web/tracker/models.py

```diff
@@ -12,8 +12,14 @@ class Contact(models.Model):
 
 #Join Table
 class OneOnOne(models.Model):
-  user = models.ForeignKey(User, to_field='id', null=True)
-  contact = models.ForeignKey(Contact, to_field='id', null=True)
+  user = models.ForeignKey(User,
+      to_field='id',
+      null=True,
+      on_delete=models.DO_NOTHING)
```

Comment:
It was complaining about not providing this value; one of the lib versions was bumped.